lib/rodauth/rails/feature/render.rb

```diff
@@ -50,6 +51,12 @@ def _view(meth, *)
           html = html.gsub(/<form(.+)>/, '<form\1 data-turbo="false">') if meth == :view
           html
         end
+
+        def set_title(title)
+          if title_instance_variable
+            _rails_controller_instance.instance_variable_set(title_instance_variable, title)
```

Comment:
Here `_rails_controller_instance` is a non-cached instance of the controller, every invocation of this method returns a new instance. We want to use the cached `rails_controller_instance` method instead (generated by `auth_cached_method :rails_controller_instance`).

Reply:
done 👍